src/main/scala/wvlet/airframe/Airframe.scala

```diff
@@ -16,11 +16,5 @@ import wvlet.log.LogSupport
 
 object Airframe extends LogSupport {
   def newDesign: Design = Design.blanc
-
-  private var _currentSession : Option[Session] = None
```

Comment:
Deprecated current session holder, since user always need to call design.newSession